From johfel@gmx.de  Mon Nov 17 15:02:47 2008
From: Johann Felix Soden <johfel@gmx.de>
Date: Sat, 15 Nov 2008 20:44:39 +0100
Subject: Staging: et131x: remove unused variable in et1310_tx.c
To: Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <1226778279.20477.2.camel@localhost>


From: Johann Felix Soden <johfel@users.sourceforge.net>

The 'pPacket' variable is assigned, but never used. Remove it.

Signed-off-by: Johann Felix Soden <johfel@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/et131x/et1310_tx.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/drivers/staging/et131x/et1310_tx.c
+++ b/drivers/staging/et131x/et1310_tx.c
@@ -1345,7 +1345,6 @@ void et131x_free_busy_send_packets(struc
 {
 	PMP_TCB pMpTcb;
 	struct list_head *pEntry;
-	struct sk_buff *pPacket = NULL;
 	unsigned long lockflags;
 	uint32_t FreeCounter = 0;
 
@@ -1358,8 +1357,6 @@ void et131x_free_busy_send_packets(struc
 		spin_unlock_irqrestore(&pAdapter->SendWaitLock, lockflags);
 
 		pEntry = pAdapter->TxRing.SendWaitQueue.next;
-
-		pPacket = NULL;
 	}
 
 	pAdapter->TxRing.nWaitSend = 0;
